e2e/runtime_scan_test.go

```diff
@@ -0,0 +1,115 @@
+// Copyright © 2021 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"gotest.tools/assert"
+	"testing"
+	"time"
+
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+
+	"github.com/openclarity/kubeclarity/api/client/client/operations"
+	"github.com/openclarity/kubeclarity/api/client/models"
+	"github.com/openclarity/kubeclarity/e2e/common"
+)
+
+func TestRuntimeScan(t *testing.T) {
+	stopCh := make(chan struct{})
+	defer func() {
+		stopCh <- struct{}{}
+		time.Sleep(2 * time.Second)
+	}()
+	f1 := features.New("assert results").
+		WithLabel("type", "assert").
+		Assess("vulnerability in DB", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
```

Comment:
What about testing a failure scan with an image that can't be scanned?

Reply:
Done